go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/vmware-tanzu/kpack-cli
 go 1.18
 
 require (
+	github.com/aliyun/credentials-go v1.2.3
```

Comment:
I can't see where this dependency is used, can you run `go mod tidy`?

Reply:
Thanks, that cleaned up a few items.